storm-server/src/main/java/org/apache/storm/DaemonConfig.java

```diff
@@ -1040,6 +1039,13 @@
     // and log an error message, so just validating this as a String for now.
     public static final String STORM_METRIC_STORE_CLASS = "storm.metricstore.class";
 
+    /**
+     * Class implementing WorkerMetricsProcessor.
```

Comment:
It might be good to mention here in the javadocs that this is intended to be run on the supervisor.